src/index.js

```diff
@@ -72,6 +86,44 @@ class Client {
     }
     return str.join("&");
   }
+
+  /**
+   * Upload a file using Git LFS Server
+   * @param {File} file 
+   * @param {String} organizationId 
+   * @param {String} datasetId 
+   * @param {String} onProgress 
+   */
+  async upload(file, organizationId, datasetId, onProgress) {
+    let prefix = `${organizationId}/${datasetId}`;
+    let ref = {name: 'refs/heads/master' };
+    let objects = [{
```

Comment:
prefer `const` instead `let`. You will only use `let` if the variable value will be changed somewhere else.